### PR TITLE
Added ratings, snackbar and grid components.

### PR DESCRIPTION
**Pull request by Hamza-777**
Please provide feedback if any and do review this PR for me..

### CODE REVIEW

**Review by Dev-AkshatGupta — APPROVED** [review]
LGTM :) just a small suggestion host it somewhere so we can also review how the code is working that's all

**Comment by Hamza-777** [comment]
> LGTM :) just a small suggestion host it somewhere so we can also review how the code is working that's all

Sure thing akshat. Will do.

File: components/grid.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+:root {
+    --grey-3: #333;
```
Comment (vrx29) [inline components/grid.css]: variable naming could be done better. Try for eg: --color-grey-3

File: components/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#snack-opener {
+    background: var(--snack-opener-shade);
+    border-radius: 8px;
+}
+
+.snackbar {
+    position: fixed;
+    bottom: 10px;
+    left: 50%;
+    transform: translateX(-50%);
```
Comment (vrx29) [inline components/snackbar.css:10]: If you are centering it on x-axis you can try margin: 0 auto;

**Review by vrx29 — COMMENTED** [review]
You are defining CSS variables in every CSS file. Try to define it in the root file. when your code size increases you might face problems if you declare variables with the same name in multiple files.

File: components/grid.css (hunk 1 above)
Comment (Hamza-777) [reply components/grid.css]: will do better.

File: components/snackbar.css (hunk 2 above)
Comment (Hamza-777) [reply components/snackbar.css:10]: Oh that totally skipped my mind.

**Comment by Hamza-777** [comment]
> You are defining CSS variables in every CSS file. Try to define it in the root file. when your code size increases you might face problems if you declare variables with the same name in multiple files.

That's a better idea, will work on it.

**Pull request merged**